tsdb/index.go

```diff
@@ -2442,6 +2454,9 @@ func (is IndexSet) MeasurementSeriesKeysByExpr(name []byte, expr influxql.Expr)
 		}
 
 		name, tags := ParseSeriesKey(seriesKey)
+		if name == nil && tags == nil {
```

Comment:
Maybe an `||` instead of `&` ?

Reply:
I think this might be the case of having `if name == nil` alone, right?

As if the series key was `nil` then that is invalid, _but_ you can have a series key with simply no tags (`nil`) and still have a valid measurement.

I'll hold off on altering this one for now, just in case I'm not understanding it properly and it'll avoid the change back/forth.